packages/webapp/src/pages/oauth/zoom.tsx

```diff
@@ -75,16 +76,18 @@ export default ZoomOauthPage;
 
 const ZoomTestContainer = ({ ualAccount }: any) => {
     const [zoomAccountJWT, setZoomAccountJWT] = useZoomAccountJWT(undefined);
+    const queryClient = useQueryClient();
 
     const resetZoomAccount = () => {
         setZoomAccountJWT(undefined);
     };
 
     const generateZoomMeetingLink = async () => {
-        const x = await publishSecretToChain(
+        const x = await encryptSecretForPublishing(
             "https://us05web.zoom.us/j/81089675368?pwd=MUFWVkowNmk1SFVLTkJ4eEtyY01ldz09",
             ualAccount.accountName,
             ["participanta", "participantb", "participantc"],
+            queryClient,
```

Comment:
Opting for the dependency-injection route here (vs. simply importing the queryClient?)

Reply:
nice catch! when I did that we didn't have the global one! (changed now!)